cmd/ilm-add.go

```diff
@@ -44,43 +44,30 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 DESCRIPTION:
-  Add a rule to the existing set of lifecycle configuration rules,
-  If a rule with ID already exists it is replaced with the new rule.
+  Add a lifecycle configuration rule.
 
 EXAMPLES:
-  1. Add rule for testbucket on s3. Both expiry & transition are entered as dates.
-     {{.Prompt}} {{.HelpName}} --id "Devices" --prefix "dev/" \
-          --expiry-date "2020-09-17" --transition-date "2020-05-01" \
-          --storage-class "GLACIER" s3/testbucket
-
-  2. Add rule for testbucket on s3. Both expiry and transition are number of days.
-     {{.Prompt}} {{.HelpName}} --id "Docs" --prefix "doc/" \
-          --expiry-days "300" --transition-days "200" \
-          --storage-class "GLACIER" s3/testbucket
-
-  3. Add rule for testbucket on s3. Only expiry is given as number of days and transition is not set.
-     {{.Prompt}} {{.HelpName}} --id "Docs" --prefix "doc/" \
-          --expiry-days "200" \
-          --tags "docformat=docx&plaintextformat=txt&exportFormat=pdf" s3/testbucket
+  1. Add expiration rule on mybucket.
+     {{.Prompt}} {{.HelpName}} --expiry-days "200" myminio/mybucket
+
+  2. Add expiry and transition date rules on a prefix in mybucket.
+     {{.Prompt}} {{.HelpName}} -expiry-date "2025-09-17" --transition-date "2025-05-01" \
+          --storage-class "GLACIER" myminio/mybucket/doc
+
+  3. Add expiry and transition days rules on a prefix in mybucket.
+     {{.Prompt}} {{.HelpName}} -expiry-days "300" --transition-days "200" \
+          --storage-class "GLACIER" myminio/mybucket/doc
```

Comment:
And it makes examples 2 and 3 inaccurate here, the description mentions a prefix, but the policy that gets added doesn't match any particular prefix.

Reply:
Ooohh, never mind, I just noticed that the prefix is now simply added after the bucket name!

Sorry for the noise.